app/components/users/UserAdd.js

```diff
@@ -0,0 +1,72 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import toastr from 'toastr';
+import * as userActions from '../../actions/userActions';
+import UserForm from './UserForm';
+
+class UserAdd extends React.Component {
+    constructor(props, context) {
+        super(props, context);
+
+        this.state = {
+            user: Object.assign({}, props.user),
+            saving: false
+        };
+
+        this.handleSave = this.handleSave.bind(this);
+    }
+
+    handleSave(user) {
+        this.setState({ saving: true });
+        this.props.actions.createUser(user)
+            .then(() => {
+                toastr.success('User created successfully');
+                this.setState({ saving: false });
```

Comment:
Thanks @AlphaGit ;) I saw there in react-world and tutorials and I like to put the suffix: `Page` for the page-level component. In this case we can rename this component to: `UserAddPage.js` and also the edit. I think it's more clear this way... And when there are not page level component, we don't put the suffix.

Do you want to do it @matiasbeckerle ?